docs/src/for-developers/variational_inference.md

```diff
@@ -11,7 +11,7 @@ Here we'll have a look at the theory behind VI, but if you're interested in how
 
 # Motivation
 
-In Bayesian inference one usually specifies a model as follows: given data ``$\{x_i\}_{i = 1}^n$``, 
+In Bayesian inference one usually specifies a model as follows: given data ``\\{x_i\\}_{i = 1}^n``, 
```

Comment:
I'm surprised `\` is escaped here but not in other places?

Reply:
Honestly I do not fully understand this either. However, to make curly braces appear in the equation, it had to be escaped. This was also true with line breaks when using the `align` mode.